### ISSUE TITLE
State Parks as Features (or not)

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2387]
As we work on data at NMMNH, we have a lot of specimens collected in State Parks and Wildlife Management Areas.

North America, United States, New Mexico, Chaves County, Bottomless Lakes State Park
North America, United States, New Mexico, Cibola County, Bluewater Lake State Park
North America, United States, New Mexico, De Baca County, Sumner Lake State Park
North America, United States, New Mexico, Dona Ana County, Leasburg Dam State Park
North America, United States, New Mexico, Eddy County, Brantley Lake State Park
North America, United States, New Mexico, Grant County, City of Rocks State Park
North America, United States, New Mexico, Luna County, Rockhound State Park
North America, United States, New Mexico, Otero County, Oliver Lee State Park
North America, United States, New Mexico, Quay County, Ute Lake State Park
North America, United States, New Mexico, Rio Arriba County, Navajo Lake State Park
North America, United States, New Mexico, Roosevelt County, Oasis State Park
North America, United States, New Mexico, San Juan County, Navajo Lake State Park
North America, United States, New Mexico, Sierra County, Caballo Lake State Park
North America, United States, New Mexico, Sierra County, Percha Dam State Park

_Originally posted by @lin-fred in https://github.com/ArctosDB/data-migration/issues/79#issuecomment-557642081_

**Comment by Jegelewicz** [comment]
See also https://github.com/ArctosDB/data-migration/issues/79#issuecomment-557754030

![image](https://user-images.githubusercontent.com/5725767/69557906-94740880-0f64-11ea-9e60-1a37321a3191.png)

**Comment by dustymc** [comment]
I think state parts are OK as features - they're "real" things for which we could have WKT etc.

I don't think they (or anything else) should contain cryptic abbreviations, which was my only objection in the other thread.

Being non-unique isn't a problem - there are ~40 "Washington County" as well. 

Feature is weird in that there's a code table, but that's mostly just a leftover of an older model - I don't see any real problem with 40 states having a "Whatever State Park" and using the one CT value as part of the definition of 40 different THINGs.

**Comment by Jegelewicz** [comment]
>Being non-unique isn't a problem - there are ~40 "Washington County" as well.
>
>Feature is weird in that there's a code table, but that's mostly just a leftover of an older model - I don't see any real problem with 40 states having a "Whatever State Park" and using the one CT value as part of the definition of 40 different THINGs.

I agree in principle, except that there are definitions for these THINGs while there are not for "Washington County". I'm going to enter them without the abbreviations, but then we will need documentation....
